### PR TITLE
fix part of #3: created seperate css file for explore page

### PR DESCRIPTION
**Pull request by nlok5923**
Refers: #3 

Changes: 
- made seperate css file for explore page.

### CODE REVIEW

**Review by tewarig — COMMENTED** [review]
Great work.. but this pull request is using a different version of few packages than the one used in this repo.

File: src/pages/explore/Explore.module.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.paper__position {
```
Comment (pranshuchittora) [inline src/pages/explore/Explore.module.css:1]: Pls use better class names & make them camelCase

Comment (nlok5923) [reply src/pages/explore/Explore.module.css]: changed classname to more semantic once + make them camelcase

**Comment by nlok5923** [comment]
@pranshuchittora sir please review this pr.

**Review by pranshuchittora — CHANGES_REQUESTED** [review]
Pls, rename the class names. Position suffix is not intuitive.

**Comment by pranshuchittora** [comment]
Pls pull the new changes

**Comment by nlok5923** [comment]
@pranshuchittora sir please review this pr.

**Pull request merged**